Makefile

```diff
@@ -1,49 +1,15 @@
-# OASIS_START
-# DO NOT EDIT (digest: a3c674b4239234cbbe53afe090018954)
-
-SETUP = ocaml setup.ml
-
-build: setup.data
-	$(SETUP) -build $(BUILDFLAGS)
-
-doc: setup.data build
-	$(SETUP) -doc $(DOCFLAGS)
-
-test: setup.data build
-	$(SETUP) -test $(TESTFLAGS)
 
 all:
-	$(SETUP) -all $(ALLFLAGS)
-
-install: setup.data
-	$(SETUP) -install $(INSTALLFLAGS)
-
-uninstall: setup.data
-	$(SETUP) -uninstall $(UNINSTALLFLAGS)
+	@dune build @install
 
-reinstall: setup.data
-	$(SETUP) -reinstall $(REINSTALLFLAGS)
+install: all
+	@dune install
 
 clean:
-	$(SETUP) -clean $(CLEANFLAGS)
-
-distclean:
-	$(SETUP) -distclean $(DISTCLEANFLAGS)
-
-setup.data:
-	$(SETUP) -configure $(CONFIGUREFLAGS)
-
-configure:
-	$(SETUP) -configure $(CONFIGUREFLAGS)
-
-.PHONY: build doc test all install uninstall reinstall clean distclean configure
-
-# OASIS_STOP
+	@dune clean
 
-VERSION=$(shell oasis query version)
```

Comment:
please keep release target

Reply:
I'm trying to find a good method of doing the same thing with Dune. As far as I can tell, there isn't a built-in way like `oasis query`

Reply:
I am ok with populating VERSION manually in Makefile

Reply:
Actually, I can make an exe that outputs the version using `dune-build-info` library. I think it's going to do the same thing as with `oasis`.

E.g., I see this as the output of `make release` after I make the change:
```
git tag -a -m "release 0.2.1" v0.2.1
git archive --prefix=murmur3-0.2.1/ v0.2.1 | gzip > murmur3-0.2.1.tar.gz
```

Does this look right?